pose_to_segments/src/utils/metrics.py

```diff
@@ -0,0 +1,22 @@
+import torch
+import numpy as np
+
+
+def frame_accuracy(probs, gold):
```

Comment:
can you please:
1. in a comment, add the expected shape of `probs` and `gold`?
2. add python typing (`frame_accuracy(probs: torch.Tensor....`)